lib/Screens/settings.dart

```diff
@@ -192,362 +245,258 @@ class SettingsPage extends StatelessWidget {
+                                        left:
+                                            MediaQuery.of(context).size.width *
+                                                0.04),
+                                    child: Row(
+                                      mainAxisAlignment:
+                                          MainAxisAlignment.spaceBetween,
+                                      children: <Widget>[
+                                        RichText(
+                                          text: TextSpan(
+                                            text: 'Mitteilungen',
+                                            style: textspanThick,
+                                            /*defining default style is optional */
+                                            children: <TextSpan>[
+                                              TextSpan(
+                                                  text:
+                                                      '  Benachrichtigungen erlauben',
+                                                  style: textspanThin),
+                                            ],
+                                          ),
+                                        ),
+                                        Switch(
+                                          value: Hive.box('storage').get(
+                                              'notificationSwitch',
+                                              defaultValue: false),
+                                          onChanged: (bool value) {
+                                            Hive.box('storage').put(
+                                                'notificationSwitch', value);
+                                          },
+                                          activeColor: Colors.lightGreen,
+                                          activeTrackColor:
+                                              Colors.lightGreenAccent,
+                                        )
+                                      ],
+                                    ),
+                                  ),
                                 ),
-                                Icon(
-                                  Icons.import_contacts_rounded,
-                                  color: Colors.lightGreen,
-                                )
+                                ResetButton(
+                                    horizontalAlertPadding:
+                                        horizontalAlertPadding,
+                                    verticalAlertPadding: verticalAlertPadding),
+                                _divider("Sitemap"),
+                                _sitemapCard(
+                                    'Anleitung ',
+                                    '  Wie funktioniert die App? ',
+                                    Icon(
+                                      Icons.article_rounded,
```

Comment:
color is always light green, only pass Icons.article_rounded to _siteMapCard and create Icon in function